src/state/pages/actions.js

```diff
@@ -354,7 +352,7 @@ const putSelectedPageStatus = status => (dispatch, getState) =>
         }
       }
       resolve();
-    });
+    }).catch();
```

Comment:
This will result in an uncaught rejection. You should pass a function to catch()